src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/spec/SpecGerritTriggerHudsonTest.java

```diff
@@ -91,6 +91,8 @@
     private SshServer sshd;
     private SshdServerMock serverMock;
     private GerritServer gerritServer;
+    private static final int DELAY = 2000;
```

Comment:
I think you had to do this because you got outside of the area for `MagicNumber` ignore rule?
You could just increase the number on Line 73.

Reply:
I don't mind moving away from that magical number unless you want it there. Either way I can revert and update the checkstyle or remove it entirely.